### ISSUE TITLE
Stray `-` 

### ISSUE THREAD

**Issue by ericawood** [biolink/biolink-model#808]
**Describe the bug**
A recent update to the annotations section of `decreases abundance of` that features a dash on line 1945 is causing issues with parsing the biolink YAML file:
https://github.com/biolink/biolink-model/blob/615629a4e3ad198b68c3ef08c4e8935816c7deb3/biolink-model.yaml#L1936-L1950

It now translates into this (JSON):
```
    "annotations": [
        {
            "tag": "biolink:canonical_predicate",
            "value": true
        }
    ],

```

**To Reproduce**
To reproduce, parse the biolink YAML file in Python and dump it into JSON.

**Expected behavior**
Here's what most of the annotations properties in the biolink YAML file look like:
https://github.com/biolink/biolink-model/blob/615629a4e3ad198b68c3ef08c4e8935816c7deb3/biolink-model.yaml#L1914-L1928
which converts into this JSON:
```
    "annotations": {
        "tag": "biolink:canonical_predicate",
        "value": true
    },
```

**Comment by cmungall** [comment]
Hi @ericawood - in fact the range of annotations is a list of annotations, so the current JSON is correct. Can you indicate why you think it's incorrect?

**Comment by sierra-moxon** [comment]
Yeah the syntax is fixed In PR 771 - I’ll check into why it didn’t fail on my previous syntax error.  This does look like a stray dash without 771.
